### ISSUE TITLE
Empty date values cause a ValueError

### ISSUE THREAD

**Issue by alandmoore** [mherrmann/pypxlib#13]
When trying to access the value of a Date column with an empty value (Null or empty string, not sure; in Paradox it just appears that the date is empty), I get the following exception:

```
  File "/home/alanm/.local/lib/python3.7/site-packages/pypxlib/__init__.py", line 256, in __getitem__
    return field.deserialize(pxval)
  File "/home/alanm/.local/lib/python3.7/site-packages/pypxlib/__init__.py", line 139, in deserialize
    return self._deserialize(pxval.value)
  File "/home/alanm/.local/lib/python3.7/site-packages/pypxlib/__init__.py", line 164, in _deserialize
    return self._deserialize_days(pxval_value.lval)
  File "/home/alanm/.local/lib/python3.7/site-packages/pypxlib/__init__.py", line 172, in _deserialize_days
    return date(year.value, month.value, day.value)
ValueError: year 0 is out of range
```

**Comment by mherrmann** [comment]
I suppose the correct behavior would be to return `None` instead. If you want to create a Pull Request for this, I'd be happy to accept it. I won't have time to implement this myself.

**Comment by mherrmann** [comment]
Should be fixed in 2.1. Thank you @alandmoore for submitting a fix!
